app/pulp/app/models/progress.py

```diff
@@ -3,13 +3,17 @@
 """
 from gettext import gettext as _
 import logging
+import datetime
 
 from django.db import models
 
 from pulp.tasking import util
 from pulp.app.models import Model, Task
+from django.utils import timezone
 
 _logger = logging.getLogger(__name__)
+# number of ms between save() calls when _using_context_manager is set
+_batch_interval = 500
```

Comment:
Can this become BATCH_INTERVAL I think python constants are usually in all caps.
Also having a line of whitespace between L#15 and L#14 would be good to group the comment w/ the line it is for more clearly.